lib/ansiblelint/utils.py

```diff
@@ -712,7 +712,7 @@ def get_playbooks_and_roles(options=None):
 
     for p in map(Path, files):
 
-        if any(str(p).startswith(file_path) for file_path in options.exclude_paths):
+        if any(str(p.resolve()).startswith(file_path) for file_path in options.exclude_paths):
```

Comment:
Looks like @cans also tried to fix it @ https://github.com/ansible/ansible-lint/pull/749/files#diff-8a54e9ace4c09bec09fd356284e1853eR859

Reply:
Maybe you could get some inspiration there?

Reply:
We need to start introduction of type checks in order to avoid so many casts in our code but this should be done in separated PR.